app/lib/actions/katello/alternate_content_source/refresh.rb

```diff
@@ -0,0 +1,27 @@
+module Actions
+  module Katello
+    module AlternateContentSource
+      class Refresh < Actions::EntryAction
+        def plan(acs)
+          action_subject(acs)
+          concurrence do
+            acs.smart_proxies.each do |smart_proxy|
+              plan_action(Pulp3::Orchestration::AlternateContentSource::Refresh,
+                          acs, smart_proxy)
+            end
+          end
+          plan_self(acs_id: acs.id)
+        end
+
+        def finalize
+          acs = ::Katello::AlternateContentSource.find_by(id: input[:acs_id])
+          acs.update(last_refreshed: ::DateTime.now)
+        end
+
+        def humanized_name
+          _("Refresh Alternate Content Source")
+        end
+      end
+    end
+  end
+end
```

Comment:
I'll have to check if that's possible

Reply:
Is it the presenter you're talking about? I suppose I'd need something like
```ruby
        def presenter
          found = all_planned_actions(Pulp3::AlternateContentSource::Refresh) if found.empty?
          Helpers::Presenter::Delegated.new(self, found)
        end
```

Reply:
Brian on Pulp said "it should be roughly the time of N on-demand syncs". Having a presenter would likely be a good idea.  Should we do that now or later? This PR should probably merge soon.

Reply:
At a glance, the presenter stuff might be a little complicated because, unlike a sync, a refresh is like multiple syncs. We would need to present each task within the task group.  I'm going to suggest that we do it in a separate PR since I have limited time to work on this one.